sale_automatic_workflow/README.rst

```diff
@@ -85,6 +85,7 @@ Contributors
 * Sodexis
 * Dave Lasley <dave@laslabs.com>
 * Akim Juillerat <akim.juillerat@camptocamp.com>
+* Thomas Fossoul <thomas@niboo.com>
```

Comment:
@tfossoul As there are readme fragments (in readme directory) add it there as it will be overwritten after merge.